wicket-core/src/main/java/org/apache/wicket/DequeueContext.java

```diff
@@ -226,9 +227,9 @@ private DequeueTagAction canDequeueTag(ComponentTag open)
 		}
 
 		DequeueTagAction action;
-		for (int i = containers.size() - 1; i >= 0; i--)
+		for (MarkupContainer container : containers)
```

Comment:
Only small side note: Is the order of the for loop important in this case?

Before:
* Last container was taken, then one before last and so on...

Now:
* First is taken, then the second and so on...

The reason why I ask is that the loop was build to explicit iterate in the opposite direction.

Reply:
I preserved all of the iteration behaviour.

The `ArrayDeque` iterates "backwards" already, hence why there's a `descendingIterator()` for a "forward" iteration.

Reply:
Ah ok, thanks a lot for the explanation. Great changes 👍

Reply:
No problem! :)